tests/test_density.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+import matplotlib.pyplot as plt
+import pint.toa as toa
+from pint.models import get_model
+from pylab import *
+from copy import deepcopy
+import io
+
+
+def test_get_highest_density_range(ndays=7 * u.d):
```

Comment:
Since we don't call test functions, there's not much point giving them arguments - or at least, we use arguments for different things (fixtures and parametrization if you're curious). So maybe just set this on the first line. Or you could run this function with twice with two different values of `ndays` using `@pytest.mark.parametrize()` to make two different tests.